src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MissingOverride/case1/context/defaultContext.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/Test.java b/Test.java
+index 0999e31..6310d55 100644
+--- a/Test.java
++++ b/Test.java
+@@ -3,19 +3,21 @@ package TreeWalker.MissingOverride;
+ public class Test {
+ 
+     /** {@inheritDoc} */
+-    @Override
+     public void test1() {  // violation without filter
+ 
+     }
+ 
++    /** {@inheritDoc} */
+     public void test2() {  // violation without filter
+ 
+     }
```

Comment:
` void test2() {} // violation without filter`

please make code compact to let us focus on what matter.

Reply:
done